pyspecProcScripts/correlation_alignment.py

```diff
@@ -172,7 +139,7 @@ def correl_align(
     energy_diff = 1.0
     i = 0
     energy_vals = []
-    this_E = (abs(s_orig.C.sum(indirect_dim)) ** 2).data.sum().item() / N**2
+    this_E = (abs(s_jk.C.sum("repeats")) ** 2).data.sum().item() / N**2
```

Comment:
add a comment -- this divides by an extre N compared to above, because if the signals along repeats are the same, then the energy of the resulting sum should increase by N (vs. taking the square and then summing, which is what we do above)